src/components/monitor/components/EmailToolBar.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import Button from '@/components/shared/Button';
+import { Info } from '@phosphor-icons/react';
+import { Tooltip } from 'react-tooltip';
+import axios from 'axios';
+import { HaveIbeenPwnedText } from '@/assets/types/have-i-been-pawned';
+
+interface EmailToolBarProps {
+  textContent: HaveIbeenPwnedText['HeroSection']['EmailToolBar'];
+  onResultChange: (result: any[]) => void;
+  onErrorChange: (error: string | null) => void;
+}
+
+function EmailToolbar({ textContent, onResultChange, onErrorChange }: Readonly<EmailToolBarProps>) {
```

Comment:
Use const instead, as this is a component. For example:
```
export const EmailToolBar = ({...}: Readonly<EmailToolBarProps>) {
....
}
```

Reply:
Done